README.md

```diff
@@ -29,7 +29,17 @@ npx cwe-tool [...command-line options...]
 
 ## Docker
 
-TBD (❌ PRs welcome)
+Build and run in one step
+
+```shell
+docker build -t owasp/cwe:sha1sum . && docker run --rm owasp/cwe:sha1sum --id 22
```

Comment:
what's the purpose of `sha1sum` in `owasp/cwe:sha1sum`  ?

Reply:
Ahh ok. Since this is mostly about a person building this locally I think using `:latest` makes sense here.